index.html

```diff
@@ -296,9 +296,9 @@
 <body>
     <section id='abstract'>
       <p>
-A [=controller document=] is a set of data that specifies one or more
-relationships between a [=controller=] and a set of data, such as a set of
-public cryptographic keys.
+A [=controller document=] contains cryptographic material and service
+endpoints for the purposes of verifying proofs from, and interacting
+with, the [=controller=] of an identifier.
```

Comment:
This is excellent, I really like it.

Reply:
I also like it, though it does feel to me like it could be more general about verification methods, as it makes reference only to concrete examples of verification methods that happen to use cryptographic material (yes, these are the most commonly used today and the ones we provide examples for).

I don't have any suggestions to address this that I think would improve it -- and this language is definitely an improvement over what we have. I also don't think how it's written precludes other forms of verification methods and it is perhaps more immediately understandable via the concrete reference than saying something more abstract.

Reply:
```suggestion
A [=controller document=] contains cryptographic material and identifies
service endpoints that can be used to verify proofs from, and interact
with, the [=controller=] of an identifier.
```